src/api/github/index.js

```diff
@@ -55,7 +55,7 @@ const githubStub = {
         },
         {
           method: 'GET',
-          path: '/repos/{org}/{repo}/git/refs/{path}',
+          path: '/repos/{org}/{repo}/git/ref/{path}',
```

Comment:
weirdly the get version uses the `/ref/` path
https://docs.github.com/en/rest/git/refs?apiVersion=2022-11-28#get-a-reference

where as the post/patch/delete etc use `/refs`
https://docs.github.com/en/rest/git/refs?apiVersion=2022-11-28#create-a-reference